conda/plugins/types.py

```diff
@@ -213,62 +212,65 @@ class CondaSetting:
     aliases: tuple[str, ...] = tuple()
 
 
-class ReporterHandlerBase(ABC):
+class ReporterRendererBase(ABC):
     """
-    Base class for all reporter handlers.
+    Base class for all reporter renderers.
     """
-
-    def render(self, data: Any, **kwargs) -> str:
+    def default(self, data: Any, **kwargs) -> str:
+        """
+        How to render the data by default
+        """
         return str(data)
 
     @abstractmethod
-    def detail_view(self, data: dict[str, str | int | bool], **kwargs) -> str:
+    def table(self, data: Any, **kwargs) -> str:
         """
-        Render the output in a "tabular" format.
+        Renders the data in a tabular format.
         """
 
     @abstractmethod
-    def envs_list(self, data, **kwargs) -> str:
+    def list(self, data: Any, **kwargs) -> str:
         """
-        Render a list of environments
+        Renders the data in a plain list
         """
```

Comment:
This is a key change in how it handles render styles, should give us some flexibility for future expansion.